test/prototype_transforms_kernel_infos.py

```diff
@@ -564,7 +597,14 @@ def sample_inputs_affine_video():
             sample_inputs_fn=sample_inputs_affine_image_tensor,
             reference_fn=pil_reference_wrapper(F.affine_image_pil),
             reference_inputs_fn=reference_inputs_affine_image_tensor,
-            closeness_kwargs=DEFAULT_PIL_REFERENCE_CLOSENESS_KWARGS,
+            closeness_kwargs={
+                (("TestKernels", "test_against_reference"), torch.float32, "cpu"): dict(
+                    atol=10 / 255, rtol=0, agg_method="mean"
+                ),
+                (("TestKernels", "test_against_reference"), torch.uint8, "cpu"): dict(
+                    atol=10, rtol=0, agg_method="mean"
+                ),
+            },
```

Comment:
A average diff of `10` is still pretty high, but far from the other problematic ones. We can still investigate though.